templates/meta_woz/dialogues/templates.yaml

```diff
@@ -0,0 +1,111 @@
+dataset: meta_woz
+subset: dialogues
+templates:
+  4b64c6e9-0aa0-431f-85b5-8367daa4773a: !Template
+    id: 4b64c6e9-0aa0-431f-85b5-8367daa4773a
+    jinja: "What does this conversation between a chatbot and a client talk about\
+      \ ? \n{{turns | join(',  ')}}\n|||\n{{domain.replace('_', ' ') | lower }}"
+    name: predict_domain_2
+    reference: ''
+    task_template: false
+  56151136-82a4-455b-98ed-aea6ee1c273d: !Template
+    id: 56151136-82a4-455b-98ed-aea6ee1c273d
+    jinja: '{% set count = namespace(value=0) %}
+
+      {% for i in range(range(turns|length) | random() - 1) %}
+
+      {{["AI Assistant", "Client"][i% 2]}}: {{turns[i]}}
+
+      {% set count.value= i + 1 %}
+
+      {% endfor %}
+
+      {{["AI Assistant", "Client"][ count.value % 2]}}:
+
+      |||
+
+      {{turns[count.value] }}
+
+
+      '
+    name: predict_random
```

Comment:
It seems like this can result in predicting a single utterance, e.g. the prompt is just "AI Assistant:". This seems too hard. Can you make sure there is at least one turn of dialog before the model has to predict anything?

Reply:
My intuition is that the model can learn to predict some statements for initializing a dialog like for example : How may I help you ? , what do you want to ask for ? , etc.

Reply:
I guess I'm just not sure this is a useful task - if, on the one hand, there are only a finite/small number of feasible generations, then you're basically just training the model to memorize the small set of things a helperbot would start a conversation with. On the other hand, if there are lots of possible "first utterances", then you're approaching unconditional generation, and we will have already trained the model a lot on unconditional generation.

Reply:
Ok, that makes sense!